### PR TITLE
Parquet fdw

### PR DESCRIPTION
**Pull request by greenape**
### I have:

- [x] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [x] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [x] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Adds parquet foreign table support to FlowDB, via parquet_fdw.

### CODE REVIEW

**Comment by greenape** [comment]
@Thingus this was mostly you, so extra good if you take a dekko to make sure I haven't mucked it up :)

File: flowdb/testdata/test_data/py/zz_convert_events_to_parquet.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    "volume_download": pyarrow.int64(),
+    "msisdn": pyarrow.string(),
+    "location_id": pyarrow.string(),
+    "imsi": pyarrow.string(),
+    "imei": pyarrow.string(),
+    "tac": pyarrow.int64(),
+    "network": pyarrow.string(),
+    "operator_code": pyarrow.int64(),
+    "country_code": pyarrow.int64(),
+}
```
[the remark this answers is not in the document]
Comment (Thingus) [reply flowdb/testdata/test_data/py/zz_convert_events_to_parquet.py:70]: EDIT: Given how it's used below, probably no real point.

**Review by Thingus — COMMENTED** [review]
I'm thinking we might want to move `zz_convert_events_to_parquet` and friends out of test_data and into flowdb/bin/build now they're being operationalised

**Comment by greenape** [comment]
> I'm thinking we might want to move `zz_convert_events_to_parquet` and friends out of test_data and into flowdb/bin/build now they're being operationalised

No, because this is only for the test data- I don't think this script would be appropriate as is for migrating etc.

Ideally we dispense with the need to convert by: replacing the existing cvs with converted parquet, and generating to parquet rather than Postgres storage for the synth data.

But as it stands, this gets us what we want right now for the test data and ensures that we can test.

**Review by Thingus — APPROVED** [review]

**Pull request merged**